ansq/tcp/reader.py

```diff
@@ -182,18 +183,17 @@ def __init__(
         self._poll_lookup_task: Optional[asyncio.Task] = None
 
         # Keep original on close callback to call it in `self._on_close_connection`
-        self._orig_on_close_callback = self._reader.connection_options.get("on_close")
+        self._reader_on_close = self._reader.connection_options.on_close
```

Comment:
```suggestion
        self._orig_on_close_callback = self._reader.connection_options.on_close
```

I read the `reader_on_close` as if the reader is closed then the callback is called (i.e., `reader.close()` is called). But that's not exactly what happens. This callback is called when any of connection from the reader is closed (i.e., `reader.connections[0].close()` is called).

I still think that the previous name is correct. Whould you like to change it to something else?

Reply:
Ok. It was an attempt to make the name a bit shorter :)

Reply:
Ah, okay. `self._orig_on_close` is short enough.

Reply:
Renamed back

Reply:
Okay, thanks!